### PR TITLE
Adding PyQt5 QML example

### PR DESCRIPTION
**Pull request by Siecje**

### CODE REVIEW

File: examples/pyqt5_qml/basicqml/main.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+                              appWindow, "JSManager");
+}
+
+function onLoad(){
+    console.log('inside');
+    var jsManager = new JSManager();
+
+    // Populate text
+    jsManager.text.connect(function(text){
+      console.log(text);
```
Comment (takluyver) [inline examples/pyqt5_qml/basicqml/main.js]: Two languages for the price of one!

**Comment by takluyver** [comment]
Thanks! So this is working? Did you get around the issue you mentioned on Reddit, which I opened as #75 ?

File: examples/pyqt5_qml/basicqml/main.js (hunk 1 above)
Comment (Siecje) [inline examples/pyqt5_qml/basicqml/main.js]: I will remove the ```console.log```.

Even if it doesn't show up in the installed application.

**Comment by Siecje** [comment]
I was able to build and install it on Windows 10.

I have not tried on Windows 7, like Steve said it might just require a restart.

File: examples/pyqt5_qml/basicqml/main.js (hunk 1 above)
Comment (takluyver) [inline examples/pyqt5_qml/basicqml/main.js]: It's not a problem, I was just amused that we're now packaging interpreters for both Python and Javascript in one installer. :-)

**Comment by takluyver** [comment]
Thanks. Hopefully the issue on Windows 7 was just the windows update package failing to install, but let me know if it happens again or you can find any more hints as to the problem.

**Pull request merged**